### ISSUE TITLE
About the backward function of crossentropy loss in deep SHAP 

### ISSUE THREAD

**Issue by JFChi** [shap/shap#698]
If we want to explain the model loss (e.g. crossentropy loss) w.r.t. the the input using deep SHAP, how do we rewrite the backward function of of the crossentropy loss?

**Comment by detrin** [comment]
@JFChi I believe the whole point of shap it is to explain the prediction, hence the shap values are bound to predictions of the model and not to a loss function.

However, I think you could make a wrapper model that would run the prediction on the underlying model and then use it to calculate the loss function. The whole wrapped model could be addressed as a regressor and you could of course use shap values to explain the regressor (your desired loss function).
